_data/docs.yml

```diff
@@ -86,7 +86,8 @@
       keywords : ["tabs","motion","brackets"]
     - title: Desk Occupancy Installation
       keywords : ["tabs","motion"]
-
+    - title: leak Detector Installation Guide
```

Comment:
I think you meant Leak with a "L" unless it will show "leak Detector Installation Guide" in the menu